docs/core_docs/docs/integrations/chat/mistral.mdx

```diff
@@ -0,0 +1,45 @@
+---
+sidebar_label: Mistral
+---
+
+import CodeBlock from "@theme/CodeBlock";
+
+# ChatMistralAI
+
+[OMistral AI](https://mistral.ai/) is a research and hosting platform for LLMs.
```

Comment:
Typo?

Reply:
whoops, copypasta from Ollama docs, forgot to drop that O